etc/build/PPS_BUILD_BUCKET_COUNT

```diff
@@ -0,0 +1 @@
+3
```

Comment:
This is where we'll specify the number of buckets to split the PPS test suite into. We also have to explicitly enumerate the corresponding environment variables in `.travis.yml`

I'm open to ways to keep this DRYer, but this seemed like the simplest implementation method

Reply:
LGTM (easy to change later, in any case)